### ISSUE TITLE
Update API to serve data based on Webflow CMS

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/app.co#791]
https://api.app.co/api/apps

The API should be updated to [get all Webflow CMS app items](https://developers.webflow.com/#get-all-items-for-a-collection) and serve those up instead of apps stored in the deprecated Next.js app's database so the content doesn't get stale over time for third parties who rely on this API.

We'll likely need to coordinate with known 3rd party implementers so they know to expect a limited set of properties per app since we've reduced the scope of data we track for apps in Webflow.

**Comment by hstove** [comment]
Reading the CMS API docs, I don't think simply using that will work. You need to have an API key, and their API keys grant full access to your account. Also, the API response is exposing a bit more information than what we'd want.

What we can do is utilize the app.co API server to shim this API and format it. This way, we don't have to worry about authentication, and we could craft the API response so that 3rd parties have as easy of an upgrade as possible.

**Comment by markmhendrickson** [comment]
Yep I had shimming in mind as well. We pull from Webflow's CMS, reformat the data however we see fit, and serve it up at the same API URL we have in place.

**Comment by hstove** [comment]
Yep. Unfortunately it means we can't quite get 100% away from developer maintenance, but it reduces the scope quite a lot. And we can shut down the other routes after some time.

Nitpick, but unless we can promise that no fields will be removed (aka no breaking changes), then I'd prefer a different URL like `/api/v2/apps`.

**Comment by markmhendrickson** [comment]
Yep agreed on the versioning. And I imagine there _may_ be an API wrapper service out there that hosts this shim for us? I'm not familiar with one exactly...but feels like something AWS would provide.
